testplan/testing/multitest/driver/base.py

```diff
@@ -100,6 +104,12 @@ class Driver(Resource):
     :type stdout_regexps: ``list`` of ``_sre.SRE_Pattern``
     :param stderr_regexps: Same with log_regexps but matching stderr file.
     :type stderr_regexps: ``list`` of ``_sre.SRE_Pattern``
+    :param file_logger: A standalone file which is placed under the `runpath`
```

Comment:
Sends driver's logging to a user specified file under `runpath`. By default, logging goes to console and top level testplan.log.